elasticdl/python/ps/main.py

```diff
@@ -16,4 +25,40 @@ def main():
     optimizer = model_module[args.optimizer]()
 
     logger.info("Starting PS pod with optimizer as %s", optimizer._name)
-    # TODO: create `Parameters` class instance, and start PS RPC servicer.
+
```

Comment:
Would it be better if the main function keeps the same style as `master.main`?

Reply:
Re-org `ps.main` to be consistent with `master.main`.